### ISSUE TITLE
False positive on cronjob-backed pods

### ISSUE THREAD

**Issue by rlex** [coroot/coroot#29]
Found interesting thing: 
I have cronjob (renovate) running inside k8s cluster. Since pod starts, performs job, and ends, it seems like coroot detects it memory usage grouping all pods as "renovate-github" in my case and reports memory leak. Screenshot attached.

Looks like it's aggregating memory usage on each cronjob pod run?

<img width="601" alt="Screenshot 2023-03-08 at 04 37 05" src="https://user-images.githubusercontent.com/84462/223596994-6d30ecfe-26cf-466d-8798-d2b280b01f74.png">

**Comment by rlex** [comment]
One thing that comes to mind is to look into metadata of pod, particulary this part:

```
metadata:
  ownerReferences:
    - apiVersion: batch/v1
      kind: Job
```

Which tells this is pod fired from job

**Comment by def** [comment]
It seems that the best solution here would be to skip this inspection for CronJobs:

* There is a high probability of false positives due to the fact that the task launch process may appear as a leak.
* Even a real leak in a short-lived process is not a problem.

**Comment by def** [comment]
This is fixed in v0.14.4
